checker/build.xml

```diff
@@ -256,6 +254,20 @@
         <delete file="${build}/.timestamp"/>
     </target>
 
+    <target name="-compile-java8" depends="-force-compile-java8"
+            description="Compiles java 8 sources if JAVA_8_HOME is set or javac is already java8">
+        <antcall target="compile-java8-sources">
```

Comment:
No, -force-compile-java8 only runs if jdk7orlower is true and compile-java8-sources only runs if jdk8orhigher.  So at most only on of the targets will run.